WordPress/twentytwentyfive
# Add optional color palettes

Resolves #24 
Added optional color palettes to the theme



**Screenshots**
<img width="293" alt="Capture d’écran 2024-08-26 à 23 24 10" src="https://github.com/user-attachments/assets/26548c4a-5222-4062-9355-50f53e3eb43f">




**Testing Instructions**

1. Activate the theme
2. Customize theme 
3. Go to styles
4. There should be 9 different palettes
